<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogineSenha</class>
 <widget class="QMainWindow" name="LogineSenha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Btn_Entrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius: 5px;
color: white;
background-color:rgb(20, 4, 4);
border: 2px solid #8f8f91;
border-style: outset;
text-align: center;
-webkit-box-shadow: 3px 3px 5px 0px rgba(0,0,0,0.75);
-moz-box-shadow: 3px 3px 5px 0px rgba(0,0,0,0.75);
box-shadow: 3px 3px 5px 0px rgba(0,0,0,0.75);}

QPushButton::hover{
	background-color: rgb(52, 52, 52);
	color:rgb(126, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Usuario">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Senha">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 9pt &quot;Yu Gothic UI bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 9pt &quot;Yu Gothic UI bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Ver_Senha">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 9pt &quot;Yu Gothic UI bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mostrar Senha</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fundo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/imgs/imagens/fundo.jpg</pixmap>
    </property>
    <property name="indent">
     <number>-10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Informcao">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#900
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>fundo</zorder>
   <zorder>Btn_Entrar</zorder>
   <zorder>txt_Usuario</zorder>
   <zorder>txt_Senha</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>Ver_Senha</zorder>
   <zorder>Informcao</zorder>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
